spec/features/accessibility_spec.rb

```diff
@@ -0,0 +1,10 @@
+# frozen_string_literal: true
+
+require "rails_helper"
+
+RSpec.describe "Accessibility tests", driver: :selenium_chrome_headless do
```

Comment:
Note these tests require the selenium web driver, and we have a new pattern (introduced below) for injecting a custom driver for a set of tests.